test/format/format.sh

```diff
@@ -415,15 +411,14 @@ resolve()
 		grep 'successful run completed' $log > /dev/null && {
 			rm -rf $dir $log
 			success=$(($success + 1))
-			verbose "$name: job in $dir successfully completed"
+			msg "job in $dir successfully completed"
```

Comment:
Would moving this print (and a few others down for successful jobs) out of verbose control flood the task log for those 6-hour Evergreen format tasks? I feel there's a reason we control the message dump using the verbose flag previously.

Reply:
I think we should consider what developers would find useful. The vast majority of the task logs are from the build output, so I don't think adding one line per format job would be a huge concern.

Reply:
@lukech, 

> Would moving this print (and a few others down for successful jobs) out of verbose control flood the task log for those 6-hour Evergreen format tasks? I feel there's a reason we control the message dump using the verbose flag previously.

It won't flood the task log. It's one additional line of output per format run, and we don't get that many format runs within an Evergreen job.